app/helpers/smart_proxies_helper.rb

```diff
@@ -1,5 +1,5 @@
 module SmartProxiesHelper
-  TABBED_FEATURES = ["Puppet","Puppet CA"]
+  TABBED_FEATURES = ["Puppet", "Puppet CA", "Logs"]
```

Comment:
we should ensure they are sorted when presented (or keep the current order) as i already saw in one of the screenshots the following tabs:
Puppet, Logs, Puppet CA which makes little sense